lib/types.ts

```diff
@@ -62,9 +62,10 @@ export enum ENVIRONMENT {
   SANDBOX = 'sandbox',
 }
 export enum REQUEST_SYNC_STRATEGIES {
-  SYNC_STRICT_TIMEOUT = 'sync-strict-timeout',
-  SYNC_SOFT_TIMEOUT = 'sync-soft-timeout',
```

Comment:
This is _technically_ a breaking change, but no one is using this.

Reply:
If we don't need to force existing users to upgrade then I think we're OK.